src/utils/view.js

```diff
@@ -61,6 +61,11 @@ export function simpleToPlotly(view) {
       }
     }
   }
+  if(view.spec.series.length === 1) {
```

Comment:
@rufuspollock Fortunately, Plotly handles it by default:
![image](https://cloud.githubusercontent.com/assets/17809581/23845216/fbbd3a6c-07f0-11e7-8a1e-fd903ecc526e.png)